### ISSUE TITLE
Uppercase and related issues in column description file

### ISSUE THREAD

**Issue by weaverba137** [desihub/desidatamodel#146]
Some of the column names in the `column_descriptions.csv` file are lower- or mixed-case. Some examples:

* `C_Halpha` 
* `elg_mask`
* `flux_g_dered` (note that this also has no type or units in the file)
* `o2c`
* `sort`
* `Z_not4clus`

This brings up several questions:

* Where do these come from?
* What does case actually signify in terms of column names?
* How is this handled in the column update code?
* Related: what is the significance of missing *type*? Does this mean that the same column has multiple types in different files?

**Comment by sbailey** [comment]
I just propagated the capitalization from the spreadsheet, which I presumed had propagated the capitalization from original data model files, which propagated them from data files on disk.  e.g. in a desidatamodel checkout:
```
$> grep C_Halpha $(find doc -name \*.rst)
doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/qso_qn-SPECTROGRAPH-TILEID-GROUPID.rst:C_Halpha             float32           Confidence line for Halpha
doc/DESI_SPECTRO_REDUX/SPECPROD/healpix/SURVEY/PROGRAM/PIXGROUP/PIXNUM/qso_qn-SURVEY-PROGRAM-PIXNUM.rst:C_Halpha             float32           Confidence line for Halpha [3]_
```
Confirming that with an example file on disk $CFS/desi/spectro/redux/fuji/tiles/cumulative/100/20210505/qso_qn-0-100-thru20210505.fits does indeed use mixed-case `C_Halpha` .

The column update code does not do lower/upper transforms when checking column names, so it needs to match the case of the column in the data file / data model.  I think it is desirable to document the columns using the case that is actually represented in the files even if FITS is formally case insensitive, since different readers handle this differently for case-sensitive languages.

Missing type: probably just means that it wasn't filled into the spreadsheet.  I don't see "flux_g_dered" in any current data model file, but perhaps it comes from some science WG product that might be added later?

**Comment by ashleyjross** [comment]
The last 5 are from LSS. (sorry!)

Z_not4clus should be transformed back to Z for any public data (the '_not4clus' is to remind people in DESI of blinding rules).

elg_mask was not used for any SV3 analysis and can be safely removed (we should). There is the (roughly) equivalent lrg_mask that was used and we should change the column name to be capitalized.

flux_g_dered ('r' and 'z') is FLUX_band/MW_TRANSMISSION_band. I think we can capitalize this. It should just be there for the BGS catalogs where we eventually get k+e corrected rest frame quantities.

o2c can be capitalized (it is the float value of the criteria used to decide good/bad redshifts for ELGs)

We should really just remove any sort column from any file.

We've yet to put anything in the public folder yet. @aureliocarnero , I think we should make these changes and any related ones, taking the SV3 EDA catalogs and writing out to somewhere like 'prep_forpublic'. Then, to be super-safe, do an np.array_equal on all of the columns, after vetting all names, etc. in the 'prep_forpublic' directory. Then, move to public directory and run the data model through, copying as much as possible from https://github.com/desihub/desidatamodel/tree/lss_eda/doc/DESI_LSS_EDA

**Comment by aureliocarnero** [comment]
@Ashley Ross ***@***.***> understood! I will make the
appropiate changes and report when it is done.
Cheers

**Comment by weaverba137** [comment]
Closed by #149.
